data/styles/clear/include/Basemap.mapcss

```diff
@@ -599,6 +601,10 @@ line|z19-[natural=earth_bank],
 {width: 11.9; dashes: 5,7,10,1.9; opacity: 0.7;}
 
 
+line|z17-[barrier=ditch],
+{width: 1.8;opacity: 1;color: @cliff;dashes: 0.9,0.9;}
```

Comment:
I think raising visibility of barrier=ditch is more justified, as its *designed* to be an obstacle, so got to be visible.

Reply:
Raised it to match `man_made=embankment` (z16-)